server/src/gql/graphql.py

```diff
@@ -0,0 +1,17 @@
+"""
+GraphQL related python types, other than MongoDB models.
+
+Keep in sync with schema.graphql (also with graphql.ts)
+"""
+# 'annotations' import needs to return enclosing type:
+# https://www.python.org/dev/peps/pep-0563/
+from __future__ import annotations
```

Comment:
doesn't look like you're returning enclosing type?

Reply:
i copy/pasted the top and forgot, my bad